### PR TITLE
docs: remove vite 2.7 notes

### PR DESCRIPTION
**Pull request by bluwy**
Remove some FAQ notes that reference Vite 2.7 as we are now already using it.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Review by ignatiusmb — APPROVED** [review]

File: documentation/faq/80-integrations.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ onMount(() => {
 
 ### How do I use Firebase?
 
-Please use Firebase SDK v9. You will also need to add Firebase dependencies to `ssr.external` ([example](https://github.com/benmccann/sveltekit-firebase/blob/9e3097fd859e4f81e4775885ecb584561f098fd3/svelte.config.js#L11)) or set `kit.ssr: false` until Vite 2.7 is released.
+Please use Firebase SDK v9. You will also need to add Firebase dependencies to `ssr.external` ([example](https://github.com/benmccann/sveltekit-firebase/blob/9e3097fd859e4f81e4775885ecb584561f098fd3/svelte.config.js#L11)).
```
Comment (bluwy) [inline documentation/faq/80-integrations.md:90]: @benmccann Is the `ssr.external` thing still needed for Vite 2.7?

File: documentation/faq/60-env-vars.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 question: How do I use environment variables?
 ---
 
-Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. Only environment variables prefixed with `VITE_` are exposed ([you can set `envPrefix` to change this](https://vitejs.dev/config/#envprefix)). For now, you would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env['YOUR_ENV_VAR']`. [Vite 2.7 will expose all environment variables on the server-side](https://github.com/vitejs/vite/pull/5404).
+Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. Only environment variables prefixed with `VITE_` are exposed ([you can set `envPrefix` to change this](https://vitejs.dev/config/#envprefix)). For now, you would need to instantiate dotenv yourself if you want all environment variables exposed in `process.env.YOUR_ENV_VAR`.
```
Comment (benmccann) [inline documentation/faq/60-env-vars.md:5]: Should we update this to differentiate between client side and server side? I think the VITE_ prefix is only needed on the client side now?

File: documentation/faq/80-integrations.md (hunk 1 above)
Comment (benmccann) [reply documentation/faq/80-integrations.md:90]: No, it's no longer needed. Firebase should hopefully just work out of the box now. We could probably just remove this entire FAQ now.

File: documentation/faq/60-env-vars.md (hunk 2 above)
Comment (Conduitry) [reply documentation/faq/60-env-vars.md:5]: We want to be careful about "environment variables that are baked in at build time" versus "server-side environment variables that are accessed at runtime". My understanding is that the `VITE_` prefix is specifically about the former, and that it is a build-time replacement performed both in server code and client code. You'd need to handle it yourself (say with `$session`) if you wanted to expose runtime server-side environment variables to the client.

File: documentation/faq/80-integrations.md (hunk 1 above)
Comment (bluwy) [reply documentation/faq/80-integrations.md]: removed the firebase section

**Comment by bluwy** [comment]
I've also updated the env var section to differentiate between build-time and runtime env vars. Using `process.env.*` as the runtime example probably isn't the best, given worker environments for example where there's no `process`. But I think this would do for now until we have a general solution for that.

**Pull request merged**